<launch>
  <param name="direct_sampling" value="true"/>
  <param name="direct_sampling_prob" value="0.1"/>
  <param name="target_tree_size" value="100"/>
  <param name="trajectory_opacity" value="0.5"/>
  <rosparam command="load" file="$(find rrtstar)/config/config.yaml"/>
  <node name="rrtstar_integrator2d_dynamic_test" pkg="rrtstar" type="rrtstar_integrator2d_dynamic_test" output="screen"/>
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find rrtstar)/rviz/int2d_config.rviz"/>
</launch>
